test/common/crudtester.go

```diff
@@ -238,10 +236,16 @@ func (c *FederatedTypeCrudTester) CheckDelete(fedObject *unstructured.Unstructur
 	name := qualifiedName.Name
 	namespace := qualifiedName.Namespace
 
+	clusterNames, err := util.GetClusterNames(fedObject)
```

Comment:
This is unnecessary.  `testClusters` is guaranteed to include all of clusters a resource could be propagated to.

Reply:
This is necessary because clusters included in testClusters but not in ClusterNames will not have orphaned resources.